roles/sap_vm_provision/tasks/platform_ansible/redhat_ocp_virt/execute_main.yml

```diff
@@ -0,0 +1,69 @@
+---
+- name: Ansible Task block for looped provisioning of KubeVirt Virtual Machines
+  any_errors_fatal: true
+  # Using environment, no_log is ineffective and log will show 'EXEC /bin/sh -c 'ENV_VAR=value python3 /AnsiballZ_ansible_module_name.py && sleep 0'
```

Comment:
I wonder if that could be avoided by setting env vars outside of the playbook?

Reply:
Why? The way I could make it work is by setting up the environment in a delegate_to statement. How would you set the environment inside the playbook? The other option I had was setting it explicitly on every call, which you didn't like either (above).